dascore/io/febus/t1utils.py

```
@@ -61,9 +66,9 @@ def _get_attrs(path, format, version):
     )
 
 
-def _scan_t1(fi: H5Reader, format, version):
+def _scan_t1(fi: H5Reader, format, version, snap=True):
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)